src/pymor/basic.py

```diff
@@ -44,11 +45,12 @@
                                         discretize_stationary_fv, discretize_instationary_fv,
                                         OnedGrid, TriaGrid, RectGrid, load_gmsh)
 from pymor.discretizers.builtin.domaindiscretizers.default import discretize_domain_default
-from pymor.discretizers.builtin.grids.boundaryinfos import EmptyBoundaryInfo, GenericBoundaryInfo, AllDirichletBoundaryInfo
+from pymor.discretizers.builtin.grids.boundaryinfos import (EmptyBoundaryInfo, GenericBoundaryInfo,
+                                                            AllDirichletBoundaryInfo)
 
-from pymor.operators.constructions import (LincombOperator, Concatenation, ComponentProjection, IdentityOperator,
-                                           ConstantOperator, ZeroOperator, VectorArrayOperator, VectorOperator,
-                                           VectorFunctional, FixedParameterOperator, AdjointOperator,
+from pymor.operators.constructions import (LincombOperator, ConcatenationOperator, ComponentProjectionOperator,
+                                           IdentityOperator, ConstantOperator, ZeroOperator, VectorArrayOperator,
+                                           VectorOperator, VectorFunctional, FixedParameterOperator, AdjointOperator,
```

Comment:
I think, we should also add the deprecated versions for now, to ensure user code that imports from `basic` does not break.